staking/programs/staking/src/lib.rs

```diff
@@ -123,6 +124,7 @@ pub mod staking {
 
         stake_account_metadata.lock = lock;
         stake_account_metadata.transfer_epoch = None;
+        stake_account_metadata.is_llc_member = false;
```

Comment:
i suggest saving the hash of the agreement they agreed to. That would e.g., let governance change the agreement text and enforce that user re-sign the new agreement. Could be useful at some point in the future.